paper/basic_training.tex

```diff
@@ -730,11 +731,170 @@ \subsubsection{Summary}
 \subsection{Barostats}
 \label{sec:barostats}
 \begin{itemize}
-\item What is it?
+\item Motivation
+\item Background
 \item Brief description of how it works
 \item Popular barostats
+\item Summary
 \end{itemize}
 
+\subsubsection{Motivation}
```

Comment:
If you don't mind, "one sentence per line" works much better with GitHub's version control. (Makes it easier to see what's edited on subsequent revisions.) May require making changes to settings for your text editor though.

Reply:
Ah, I had totally misunderstood. That makes much more sense and ill take care of that.